cns/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Use a minimal image as a parent image
+FROM ubuntu:19.10
+ARG CNS_BUILD_DIR
+
+# Update
+RUN apt-get update
+
+# Install plugin
+COPY $CNS_BUILD_DIR/azure-cns /usr/bin
```

Comment:
can we do it in another location instead of /usr/bin because im adding config file and it should be in same directory as that of azure-cns. Can we copy this to /usr/local dir? 
also we have to copy gz file and extract that. If tomorrow we have more than 1 file then we have to edit this docker file everytime

Reply:
moved